views/tool-page-base.html

```diff
@@ -26,16 +32,6 @@ <h6>Publications</h6>
             </div>
         {% endif %}
 
-        {% if doc.slides %}
-        <div class="pubs">
-            <h6>Slides</h6>
-            <ol>
-            {% for item in doc.slides %}
-                <li><a target="_blank" href={{item.link}}>{{item.name}}</a></li>
```

Comment:
Shouldn't name actually be the title of the slide deck?

Reply:
done